lib/metasploit/framework/data_service/remote/http/remote_db_export_data_service.rb

```diff
@@ -0,0 +1,13 @@
+require 'metasploit/framework/data_service/remote/http/response_data_helper'
+
+module RemoteDbExportDataService
+  include ResponseDataHelper
+
+  DB_EXPORT_API_PATH = '/api/v1/db_export'
+
+  def run_db_export(opts)
+    response = json_to_hash(self.get_data(DB_EXPORT_API_PATH, nil, opts))
+
+    process_file(response[:db_export_file], "#{opts[:path]}")
```

Comment:
Why is the `#{}` operator being used here instead of simply passing `opts[:path]`?

Reply:
It used to be needed, but I didn't change it when I took out some strings. I'll fix it.